packages/driver/src/cypress.ts

```diff
@@ -704,8 +708,8 @@ class $Cypress {
     }
   }
 
-  backend (eventName, ...args) {
-    return new Promise((resolve, reject) => {
+  backend (eventName, ...args): Promise<any> {
+    return new Promise<any>((resolve, reject) => {
```

Comment:
any reason we are adding the `any` here? I wonder if we want to make this generic which might improve type inference.